developed2019.md

```diff
@@ -100,7 +100,7 @@ If you need anything else please let one of the volunteers know.
 
 DevelopEd is meant to be a welcoming and connecting event, so please, introduce yourself and share your skills and passions with those around you! Try to be like pacman and leave room in your circle for someone new to join!
 
-Part of the society's mission is to build a diverse and welcoming developer community in Edmonton. DevelopEd, and all Dev Edmonton Society events, provide a safe and harrasment free environment to all attendees. If you've questions please read our complete [Code of Conduct](/code_of_conduct.html), speak with one of the event organizers, or reach out to [conduct@devedmonton.com](mailto:conduct@devedmontoncom).
+Part of the society's mission is to build a diverse and welcoming developer community in Edmonton. DevelopEd, and all Dev Edmonton Society events, provide a safe and harrasment free environment to all attendees. If you've questions please read our complete [Code of Conduct](/code_of_conduct.html), speak with one of the event organizers, or reach out to [conduct@devedmonton.com](mailto:conduct@devedmonton.com).
```

Comment:
Did you also want the developed@devedmonton.com email address linked up with a mailto?

Reply:
Sure! 👍